producer.py

```diff
@@ -36,43 +40,29 @@
 
 from thoth.common import __version__ as __common__version__
 from thoth.storages import __version__ as __storages__version__
-from thoth.python import __version__ as __python__version__
+from thoth.messaging import __version__ as __messaging__version__
+from version import __version__
 
-__version__ = "0.6.1"
-__service_version__ = f"{__version__}+\
-    storages.{__storages__version__}.\
-        common.{__common__version__}.\
-            python.{__python__version__}"
+from thoth.messaging import MessageBase
+from thoth.messaging.package_releases import PackageReleaseMessage
 
 init_logging()
+app = MessageBase().app
 
-_LOGGER = logging.getLogger("thoth.package_releases")
-_LOGGER.info(f"Thoth package-release-job v%s", __service_version__)
+_LOGGER = logging.getLogger("thoth.package_releases_job")
+__service_version__ = f"{__version__}+storages.{__storages__version__}.common.{__common__version__}.messaging.{__messaging__version__}"  # noqa: E501
+_LOGGER.info(f"Thoth-package-releases-job-producer v%s", __service_version__)
 
-_THOTH_METRICS_PUSHGATEWAY_URL = os.getenv("PROMETHEUS_PUSHGATEWAY_URL")
 _THOTH_DEPLOYMENT_NAME = os.environ["THOTH_DEPLOYMENT_NAME"]
+COMPONENT_NAME = "thoth-package-releases-job"
 
-prometheus_registry = CollectorRegistry()
-_METRIC_PACKAGES_NEW_AND_ADDED = Gauge(
-    "packages_added", "Packages newly added", registry=prometheus_registry
-)
-_METRIC_PACKAGES_NEW_AND_NOTIFIED = Gauge(
-    "packages_notified",
-    "Packages newly added and notification sent",
-    registry=prometheus_registry,
-)
-_METRIC_PACKAGES_RELEASES_TIME = Gauge(
-    "package_releases_time",
-    "Runtime of package releases job",
-    registry=prometheus_registry,
-)
+async_tasks = []
```

Comment:
@KPostOffice I created this variable out of the scope, so to use it inside [`package_release_update`](https://github.com/thoth-station/package-releases-job/pull/449/files#diff-4252fdf2230c8979fc9fbb96ab9bddb7R142) and also to use it inside [main](https://github.com/thoth-station/package-releases-job/pull/449/files#diff-4252fdf2230c8979fc9fbb96ab9bddb7R350)